autogpt_platform/backend/backend/blocks/pinecone_query.py

```diff
@@ -0,0 +1,50 @@
+from pinecone import Pinecone
+
+from backend.data.block import Block, BlockCategory, BlockOutput, BlockSchema
+from backend.data.model import BlockSecret, SchemaField, SecretField
+
+
+class PineconeQueryBlock(Block):
+    class Input(BlockSchema):
+        api_key: BlockSecret = SecretField(
+            key="pinecone_api_key", description="Pinecone API Key"
+        )
```

Comment:
put the two blocks in the same file for jina and pinecone so you have:
- `jina.py`
- `pinecone.py`

and even perhaps a vector db folder around them

Reply:
ill group them by use, so: 

vectordb
chunking
embeddings

This makes more sense in this case

Reply:
I understand where you're coming from, but grouping by provider is the typical use so far